spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -229,29 +229,26 @@ Following is a sample template to provision the server on PCFDev.
 ----
 ---
 applications:
-- name: {PREFERRED NAME OF THE SERVER APP}
-  host: {PREFERRED HOST}
-  memory: {PREFERRED MEMORY}
-  disk_quota: {PREFERRED DISKQUOTA}
-  timeout: {PREFERRED API TIMEOUT}
-  instances: {NO OF INSTANCES}
+- name: data-flow-server
+  host: data-flow-server
+  memory: 2G
+  disk_quota: 2G
+  instances: 1
   path: {ABSOLUTE PATH TO SERVER UBER-JAR}
   env:
-    SPRING_APPLICATION_NAME: {PREFERRED NAME OF THE SERVER APP}
+    SPRING_APPLICATION_NAME: data-flow-server
     SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_URL: https://api.local.pcfdev.io
     SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_ORG: pcfdev-org
     SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_SPACE: pcfdev-space
     SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_DOMAIN: local.pcfdev.io
     SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_USERNAME: admin
     SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_PASSWORD: admin
-    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_STREAM_SERVICES: rabbit,redis
+    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_STREAM_SERVICES: rabbit
     SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_TASK_SERVICES: mysql
     SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_SKIP_SSL_VALIDATION: true
-    SPRING_APPLICATION_JSON {"maven": { "remote-repositories": { "repo1": { "url": "https://repo.spring.io/libs-snapshot"} } } }
-    JAVA_OPTS: '-Dlogging.level.cloudfoundry=DEBUG'
+    MAVEN_REMOTE_REPOSITORIES_REPO1_URL: https://repo.spring.io/libs-release
```

Comment:
I thought that way of doing happened to work "by accident" (see https://github.com/spring-cloud/spring-cloud-dataflow-server-cloudfoundry/pull/306)